scripts/commit_classifier.py

```diff
@@ -135,6 +135,8 @@ def __init__(
         method_defect_predictor_dir: str,
         use_single_process: bool,
         skip_feature_importance: bool,
+        phabricator_deployment: str,
+        diff_id: int,
```

Comment:
The type hints here don't match the original ones:
https://github.com/mozilla/bugbug/blob/7a5e635999995b0ae603cdc960dfcdd06135358c/scripts/commit_classifier.py#L585-L586

Reply:
well noted, Sir